src/App.js

```diff
@@ -19,9 +20,12 @@ function App() {
   }, [isLoading, dispatch]);
 
   return (
-    <main>
-      <Navbar />
-      <Router />
+    <main className={styles['Bookstore-CMS']}>
+      <div className={styles['panel-bg']}>
+        <Navbar />
+        <div className={styles.Line} />
+        <Router />
+      </div>
     </main>
```

Comment:
- [optional] please kinldy add information to your readme on how to use the gTIYlXnkFso7EUewn5LZ key into the /src/key.js file. This will help a user in using the app.

Reply:
Hi @tufoinnkuo10, thanks for the advice, I edited your comment to remove the private ky.